net/rpc/server.go

```diff
@@ -3,126 +3,126 @@
 // license that can be found in the LICENSE file.
 
 /*
-	Package rpc provides access to the exported methods of an object across a
-	network or other I/O connection.  A server registers an object, making it visible
-	as a service with the name of the type of the object.  After registration, exported
-	methods of the object will be accessible remotely.  A server may register multiple
-	objects (services) of different types but it is an error to register multiple
-	objects of the same type.
+Package rpc provides access to the exported methods of an object across a
```

Comment:
Would you mind if I manually undid the whitespace changes your editor made? I usually wouldn't care, it's just that as this is a fork of a package from the standard library it's helpful to keep the diffs small to easily see what we've changed.

Reply:
+1

Reply:
I reverted the whitespace changes and learned a bit about how vscode's go extension works wrt doing these sorts of magic things.